src/bin/upgrade_octree.rs

```diff
@@ -97,6 +95,7 @@ fn main() {
             9 => upgrade_version9(&args.directory, meta),
             10 => upgrade_version10(&args.directory, meta),
             11 => upgrade_version11(&args.directory, meta),
+            12 => upgrade_version12(&args.directory, meta),
             other if other == point_viewer::CURRENT_VERSION => {
```

Comment:
maybe there should be point cloud instead of octree, because from version 12 also a s2 point cloud are possible

Reply:
Good point, I changed it.